.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+#python code owners
+ucp/            @rapidsai/ucxpy-python-codeowners
+tests/          @rapidsai/ucxpy-dask-codeowners
```

Comment:
Sorry if I'm missing something. Should this be `@rapidsai/ucxpy-python-codeowners` too?